drivers/wifi/eswifi/eswifi_socket_offload.c

```diff
@@ -0,0 +1,391 @@
+/*
+ * Copyright (c) 2019 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_LEVEL CONFIG_WIFI_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(wifi_eswifi_socket_offload);
```

Comment:
Just do `LOG_MODULE_REGISTER(wifi_eswifi, CONFIG_WIFI_LOG_LEVEL);` and remove the LOG_LEVEL define.